particle-iot/particle-cli
# feature/sc-126222/update-the-particle-cli-executable-in-workbench

## Description
This PR will:
* Use Github Actions for ci/cd and remove circleci configuration
* Fix issues with SSO login on linux systems without a display (i.e., servers)
* Fix update-cli issues that prevent to be called in background
* Remove arch and platform mappings by using the default `os.platform` and `os.arch` from nodejs
* Use `pkg.version` to set the manifest version instead of the tag



## How to Test
To check Github Actions:
* Check the generated pipelines from `staging` and also pipelines from every commit.

To test cli improvements:
* Download the generated build and use it (run the follow commands):
    * `particle version`
    * `particle update-cli`
    * `particle flash --local tinker`
    * `particle serial monitor`
    * `particle usb list`
    * `particle list`



    



## Related Issues / Discussions




## Completeness
